fcrepo-import/src/main/java/org/fcrepo/importer/Importer.java

```diff
@@ -117,18 +117,20 @@ private void importFile(final File f) throws Exception {
                 response = importContainer(uri, sanitize(model));
             }
 
-            if (response.getStatusCode() > 204) {
+            if (response.getStatusCode() == 401) {
+                throw new AuthenticationRequiredRuntimeException();
+            } else if (response.getStatusCode() > 204 || response.getStatusCode() < 200) {
                 logger.warn("Error while importing {} ({}): {}",
                    f.getAbsolutePath(), response.getStatusCode(), IOUtils.toString(response.getBody()));
             } else {
                 logger.info("Imported {}: {}", f.getAbsolutePath(), uri);
             }
         } catch (FcrepoOperationFailedException ex) {
-            throw new Exception("Error importing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
+            throw new RuntimeException("Error importing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
         } catch (IOException ex) {
-            throw new Exception("Error reading or parsing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
+            throw new RuntimeException("Error reading or parsing " + f.getAbsolutePath() + ": " + ex.toString(), ex);
         } catch (URISyntaxException ex) {
```

Comment:
Actually, someone is throwing it:
- https://github.com/fcrepo4-labs/fcrepo-import-export/pull/26/files#diff-d48f0429a55f2afa5045356359360d80R111
- https://github.com/fcrepo4-labs/fcrepo-import-export/pull/26/files#diff-d48f0429a55f2afa5045356359360d80R115

If you use `URI.create()` you get rid of the checked exception, but still get the same runtime exception detail:
https://docs.oracle.com/javase/7/docs/api/java/net/URI.html#create(java.lang.String)